Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -384,36 +408,62 @@ - (void)loadAdViewAdForParameters:(id<MAAdapterResponseParameters>)parameters ad
     
     if ( ![VungleAds isInitialized] )
     {
-        [self log: @"Vungle SDK not successfully initialized: failing %@ ad load...", adFormatLabel];
-        [delegate didFailToLoadAdViewAdWithError: MAAdapterError.notInitialized];
-        
-        return;
-    }
-    
-    [self updateUserPrivacySettingsForParameters: parameters];
-    
-    if ( isNative )
-    {
-        self.nativeAdViewDelegate = [[ALVungleMediationAdapterNativeAdViewDelegate alloc] initWithParentAdapter: self
+        [self initializeWithParameters:(id<MAAdapterInitializationParameters>)parameters completionHandler:^(MAAdapterInitializationStatus initializationStatus, NSString * _Nullable errorMessage) {
+            if ( initializationStatus == MAAdapterInitializationStatusInitializedFailure ) {
+                [self log: @"Vungle SDK not successfully initialized: failing %@ ad load...", adFormatLabel];
+                [delegate didFailToLoadAdViewAdWithError: MAAdapterError.notInitialized];
+            } else if ( initializationStatus == MAAdapterInitializationStatusInitializedSuccess ) {
+                if ( isNative )
```

Comment:
Agree with @stanley-vungle, it also needs other Ad types: AppOpenAd, Rewarded, Banner(L416 - L438 is same with L444-L466) and Native.Thanks a lot. :)

Reply:
updated